cmp/tab/TabModel.js

```diff
@@ -64,10 +67,11 @@ export class TabModel {
     }) {
         throwIf(showRemoveAction && XH.isMobileApp, 'Removable Tabs not supported in Mobile toolkit.');
 
-        this.id = id;
+        this.id = id.toString();
```

Comment:
how did this come up?

Reply:
In the dynamic tabs toolbox example, tab ids were ints. When we extract the tab id from the tab element's `data-tabId` attribute it's always a string, so we couldn't match it. TabModel id is documented as a string, and I don't think numeric ids are a realistic use case, so I thought it fine to cast to string here.